### PR TITLE
linters fix

### PR DESCRIPTION
**Pull request by AzizRAA**
In this branch we created awesome-books basic functionality with plain JS

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+      const removeBtn = document.createElement('button');
+      removeBtn.textContent = 'Remove';
+      removeBtn.classList.add('remove');
+      displaySection.appendChild(removeBtn);
+
+      const deleteBtn = document.querySelector('.remove');
+      deleteBtn.addEventListener('click', () => {
+        deleteBook(i);
+      });
```
Comment (Munsa1) [inline script.js:36]: - [ ] The correct book is not being deleted. I entered in `4` authors and books and can't delete the second, third and fourth books individually. I have to start by deleting the first book, the second, the third, and then finally the fourth book. Kindly fix this behavior,

image

- [ ] In the entries above, the remove button for `Title 2`  does not work, The remove button for `Title 1` works but deletes `Title 2` and not `Title 1`, Kindly fix this issue as well.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @AzizRAA  @Daron976 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (danmainah) [reply script.js:36]: - [ ] @team as the previous review noted, you can only remove items following a certain order, I would suggest you use index to delete a particular book and also use the array filter method 
`const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
`

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Team, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**